tools/packaging/packaging.psm1

```diff
@@ -2294,6 +2685,8 @@ function CleanupGeneratedSourceCode
         '[Microsoft.PowerShell.Commands.SetStrictModeCommand.ArgumentToPSVersionTransformationAttribute]'
         '[Microsoft.PowerShell.Commands.HttpVersionCompletionsAttribute]'
         '[System.Management.Automation.ArgumentToVersionTransformationAttribute]'
+        '[Microsoft.PowerShell.Commands.InvokeCommandCommand.ArgumentToPSVersionTransformationAttribute]'
+        '[Microsoft.PowerShell.Commands.InvokeCommandCommand.ValidateVersionAttribute]'
         )
```

Comment:
This PR needs to be backported, please move these changes to another PR

Reply:
@TravisEz13   It is only two lines, and I need the changes in order to test.  I'll create a separate PR for these two lines, but in the meantime I think this PR can be approved.

Reply:
https://github.com/PowerShell/PowerShell/pull/16800